client/src/js/modules/shipment/views/dewarstats.js

```diff
@@ -141,17 +136,8 @@ define(['marionette',
                 backgrid: { emptyText: 'No dewar stats found' } 
             }))
 
-            this.listenTo(this.countries, 'sync', this.plotMap)
-            this.plotMap()
         },
 
```

Comment:
We still need this call to `plotYears`

```suggestion

        onShow: function() {
            this.listenTo(this.run, 'sync', this.plotYears)
        },
```